tests/test_project/anexia_monitoring

```diff
@@ -0,0 +1 @@
+../../anexia_monitoring
```

Comment:
This should not be necessary, you can run `pip install -e .` to avoid this. For CI this is already in the requirements.txt

Reply:
commited on accident. thanks for the pip tip.